.travis.yml

```diff
@@ -32,8 +33,10 @@ matrix: # Add a separate config to the matrix, using clang as compiler
       env: ROS_REPO=ros-shadow-fixed
            BEFORE_DOCKER_SCRIPT="source moveit_kinematics/test/test_ikfast_plugins.sh"
 
+  fast_finish: true  # finish, even if allow-failure-tests still running
   allow_failures:
     - env: TEST=catkin_lint  # need to fix catkin_lint errors
+    - env: TEST=clang-tidy-fix  # need to fix clang-tidy issues
```

Comment:
Having this finishing being optional sounds fine to me. You mentioned that it takes some time to run - what's the order of magnitude on that? Is it seconds? Minutes?

Reply:
In the [last build](https://travis-ci.org/ros-planning/moveit/builds/488453595?utm_source=github_status&utm_medium=notification), clang-tidy-fix ran for additional 40min. Currently, it produces so many warnings, that Travis refuses to display the whole log.

Reply:
Wow.... definitely support making this optional then. Thanks.

Reply:
Me too :wink: